extensions/tajo/tajo_env.sh

```diff
@@ -0,0 +1,135 @@
+# Copyright 2014 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file contains environment-variable overrides to be used in conjunction
+# with bdutil_env.sh in order to deploy a Hadoop cluster with HBase installed
+# and configured.
+# Usage: ./bdutil -e tajo deploy
+
+# URIs of Tajo tarball to install.
+# Recommended Tajo version: Apache Tajo 0.11.0 or higher (eg. gs://your_bucket/tajo-x.xx.x.tar.gz)
+TAJO_TARBALL_URI='gs://tajo_release/tajo.tar.gz'
+
+# Tajo root directory
+TAJO_ROOT_DIR="gs://${CONFIGBUCKET}/tajo"
+
+# Tajo heap memory
+# Default is 'total_memory - 1024' (MB)
+MASTER_HEAPSIZE=
+WORKER_HEAPSIZE=
+
+# Task min memory of Tajo worker
+# Default is 3000 (MB)
+WORKER_RESOURCE_MEMORY=
+
+# Tajo meta store (catalog) setting
+# To reuse existing cloudSQL catalog, set the instance id of cloudSQL instance.
+# Or if you set non-existing instance id, new cloudSQL instance will be created automatically. 
+# (In this case, the instance id should be unique and CLOUD_SQL_CON_ID should be 'root'.)
+# To use default built-in Derby database, leave it blank.
+CLOUD_SQL_INSTANCE_ID=''
+
+# connection parameters in case of using cloudSQL for Tajo meta store
+CLOUD_SQL_CON_ID='root'
+CLOUD_SQL_CON_PW='tajo'
+CLOUD_SQL_CON_DB='tajo'
+
+# Service account setting for cloudSQL
+GCE_SERVICE_ACCOUNT_SCOPES+=('sql-admin','sql','cloud-platform')
+
+INSTALL_JDK_DEVEL=true
+
+# Tajo will be installed in this directory on each VM
+TAJO_INSTALL_DIR='/home/hadoop/tajo-install'
```

Comment:
Instead of allowing hadoop1 scripts to run in the event that a user has not supplied a -e hadoop2, consider using import_env hadoop2_env.sh (see flink_env for an example).